packages/framework/src/Resources/views/Admin/Form/Product/Price/ProductPricesWithVatSelectType.html.twig

```diff
@@ -0,0 +1,26 @@
+{% block product_prices_with_vat_select_row %}
+    <div class="form-line__box js-entity-url-list-domain-{{ domainId }}">
```

Comment:
I believe the `js-entity-url-list-domain` css class should not be here

Reply:
removed. Thanks, i did not realize that